pipelines/src/main/scala/com/fulcrumgenomics/str/pipelines/DuplexConsensusPipeline.scala

```diff
@@ -0,0 +1,122 @@
+ @arg(flag="M", minElements=1, maxElements=3, doc="The minimum number of reads supporting a consensus base/read.")
+ val minReads: Seq[Int] = Seq(6,3,3),
+ @arg(flag="E", minElements=1, maxElements=3, doc="The maximum raw-read error rate across the entire consensus read.")
+ val maxReadErrorRate: Seq[Double] = Seq(0.05),
+ @arg(flag="e", minElements=1, maxElements=3, doc="The maximum error rate for a single consensus base.")
+ val maxBaseErrorRate: Seq[Double] = Seq(0.1),
+ @arg(flag="n", doc="Maximum fraction of no-calls in the read after filtering.")
+ val maxNoCallFraction: Double = 0.2,
+ @arg(flag="f", doc="The minimum allele frequency for variant calling") val minimumAf: Double = 0.0001
+)
+  extends Pipeline(Some(output.getParent)) {
+
+  Io.assertReadable(unmappedBam)
+  Io.assertCanWriteFile(output, parentMustExist=false)
+
+  override def build(): Unit = {
+    // Files that we're going to create
+    val dir = output.getParent
+    val pre = output.getFileName
+    val mappedRaw   = dir.resolve(pre + ".raw.aligned.bam")
+    val dedupedRaw  = dir.resolve(pre + ".raw.deduped.bam")
+    val grouped     = dir.resolve(pre + ".grouped.bam")
+    val familySizes = dir.resolve(pre + ".family_sizes.txt")
+    val dsUnmapped  = dir.resolve(pre + ".consensus.unmapped.bam")
+    val dsMapped    = dir.resolve(pre + ".consensus.mapped.bam")
+    val dsFiltered  = dir.resolve(pre + ".consensus.filtered.bam")
+
+    // Copy the intervals to the directory and make a BED file
+    val regionsIl     = dir.resolve("calling_regions.interval_list")
+    val regionsBed    = dir.resolve("calling_regions.bed")
+    val copyIntervals = new ShellCommand("cp", this.intervals.toString, regionsIl.toString)
+    val makeBed       = new IntervalListToBed(intervals=regionsIl, bed=regionsBed)
+    root ==> copyIntervals ==> makeBed
+
+    // Make the grouped and consensus BAMs
+    val bwaRaw    = Bwa.bwaMemStreamed(unmappedBam=unmappedBam, mappedBam=mappedRaw, ref=ref, samToFastqCores=0, bwaMemMemory="6G")
+    val markDupes = new MarkDuplicates(inputs=Seq(mappedRaw), out=Some(dedupedRaw), templateUmiTag=Some(umiTag))
+    val group     = new GroupReadsByUmi(in=mappedRaw, out=grouped, familySizeOut=Some(familySizes),
+      rawTag=Some(umiTag), minMapQ=Some(minMapQ), strategy=AssignmentStrategy.Paired, edits=Some(edits))
+
+    val callConsensus = new CallDuplexConsensusReads(
+      in=grouped,
+      out=dsUnmapped,
+      readNamePrefix=None,
+      errorRatePreUmi=errorRatePreUmi,
+      errorRatePostUmi=errorRatePostUmi,
+      minInputBaseQuality=minInputBaseQuality
+    )
+
+    val bwaDs    = Bwa.bwaMemStreamed(unmappedBam=dsUnmapped, mappedBam=dsMapped, ref=ref, samToFastqCores=0, bwaMemMemory="6G", mergeBamAlignmentMem="6G")
+
+    val postPipeline = new PostDuplexConsensusPipeline(input=dir, truthVcf=truthVcf, ref=ref, output=dir,
```

Comment:
I might choose to move the FilterConsensusReads out from the sub-pipeline to here.  Then the sub-pipeline could take any BAM of aligned reads and generate metrics and calls - e.g. it could be run on the raw or deduped reads too if that becomes useful for debugging/comparisons.